### PR TITLE
Master

### PR DESCRIPTION
**Pull request by Franklyn-dotcom**
Added the rollback and secrets pages using markdown.

### CODE REVIEW

File: src/content/docs/guides/instant-rollback.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
+---
+title: Instant Rollback
+description: A guide about using Fly Instant Rollback to quickly revert to a previous production release.
```
Comment (pmbanugo) [inline src/content/docs/guides/instant-rollback.md]:
```suggestion
description: How to use Instant Rollback to quickly revert to a previous release.
```

File: src/content/docs/guides/instant-rollback.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+sidebar:
+  label: Use Fly Instant Rollback
```
Comment (pmbanugo) [inline src/content/docs/guides/instant-rollback.md]:
```suggestion
```

I think the title is short enough to be used here

File: src/content/docs/guides/instant-rollback.md [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+2. Click the **Instant Rollback** button. This should bring up a confirmation dialog with details about the release and a button to confirm you want to roll back. You can see information about the commit associated to the release, when it was deployed, and other information.
+
+![Rollback dialog](../../../assets/images/rollback-dialog.webp)
+
+3. Click the **Continue** button if you want to proceed. This will trigger an instant rollback event and update the [GitHub deployment](deployment-log.md) status as it progresses.
+
+:::note
+This feature is only available to [premium](pricing.md) users. If you’re on the free plan, the button will be disabled and FlyCD won’t collect information to enable it to perform instant rollback for that project.
+Check out the [pricing page](pricing.md) for information on how to subscribe to the premium plan.
+:::
```
Comment (pmbanugo) [inline src/content/docs/guides/instant-rollback.md:30]: 😎

File: src/content/docs/guides/integrating-with-fly-postgres.md [hunk 4]
```diff
@@ -0,0 +1,32 @@
+---
```
Comment (pmbanugo) [inline src/content/docs/guides/integrating-with-fly-postgres.md]: This is the page from the last PR. Please update your main branch and use a new branch for this PR so we can avoid having old commits or files already committed previously in this one

File: src/content/docs/guides/managing-secrets.md [hunk 5]
```diff
@@ -0,0 +1,52 @@
+---
+title: Managing Secrets in a project using FlyCD
```
Comment (pmbanugo) [inline src/content/docs/guides/managing-secrets.md]:
```suggestion
title: Manage Project Secrets
```

File: src/content/docs/guides/managing-secrets.md [hunk 6]
```diff
@@ -0,0 +1,52 @@
...
+description: A guide about using Fly Secrets for managing encrypted key/value pairs.
```
Comment (pmbanugo) [inline src/content/docs/guides/managing-secrets.md]:
```suggestion
description: How to manage secrets for a project.
```

File: src/content/docs/guides/managing-secrets.md [hunk 7]
```diff
@@ -0,0 +1,52 @@
...
+sidebar:
+  label: Use Fly Secrets
```
Comment (pmbanugo) [inline src/content/docs/guides/managing-secrets.md]:
```suggestion
```

File: src/content/docs/guides/managing-secrets.md [hunk 8]
```diff
@@ -0,0 +1,52 @@
...
+![A project’s secrets settings page](../../../assets/images/secrets-existing%20app.webp)
+A project’s secrets settings page
+
+You can add new secrets for production or preview apps via the **Secrets** settings page.
+
+You also see the list of secrets for the project, and the configured environment. The secret list below the form only shows the secret key, not the value. The values are only available in the continuous deployment pipeline during application deployment.
+
+There’s no separate form to edit/update secrets at the moment. The same form is used to add new secrets or edit existing ones. To change the value of an existing secret, you can add it through the form using the same key, and the value will be overwritten when it’s saved.
+
+:::note
```
Comment (pmbanugo) [inline src/content/docs/guides/managing-secrets.md:33]:
```suggestion
:::tip
```

I think a tip would best communicate the intent here

**Review by pmbanugo — COMMENTED** [review]
Well done, thanks for the changes. Please review my comments/suggestions

File: src/content/docs/guides/integrating-with-fly-postgres.md (hunk 4 above)
Comment (Franklyn-dotcom) [reply src/content/docs/guides/integrating-with-fly-postgres.md]: Okay, Understood.

File: src/content/docs/guides/deployment-env-variable.md [hunk 9]
```diff
@@ -0,0 +1,19 @@
+---
+title: Deploy Env Variable
+description: How to set environmental variable
```
Comment (pmbanugo) [inline src/content/docs/guides/deployment-env-variable.md]:
```suggestion
title: Deployment Environment Variables
description: information about FlyCD system environment variables
```

**Pull request closed without merging**